checkout/forms.py

```diff
@@ -0,0 +1,47 @@
+from django import forms
+from .models import Order
+
+class OrderForm(forms.ModelForm):
+	class Meta:
+		model = Order
+		fields = (
+			'full_name',
+			'email',
+			'phone_number',
+			'street_address1',
+			'street_address2',
+			'town_or_city',
+			'postcode',
+			'country',
+			'county',
+		)
+
+	def __init__(self, *args, **kwargs):
+		"""
+		Add placeholders and classes, remove auto-generated
+		labels and set autofocus on first field
+		"""
+		super().__init__(*args, **kwargs)
+		placeholders = {
+			'full_name': 'Full Name',
+			'email': 'Email Address',
+			'phone_number': 'Phone Number',
+			'country': 'Country',
+			'postcode': 'Postal Code',
+			'town_or_city': 'Town or City',
+			'street_address1': 'Street Address 1',
+			'street_address2': 'Street Address 2',
+			'county': 'County',
+		}
+
+		for field in self.fields:
+			if field == 'full_name':
```

Comment:
I don't feel this condition belongs in the loop - you could just do this before the loop, right?

Reply:
You mean just set autofocus on the full_name field e.g. `self.fields['full_name'].widget.attrs['autofocus'] = True` outside that loop? If so yeah that would work. Is there a benefit to doing it that way vs like this? I think I tend to mentally compartmentalize all conditions for a given loop with the loop itself e.g. "for all these things, if this, do that, if this, do that, if this, do that, otherwise do default" but I'm not opposed to separating it. It would work either way